pkgs/applications/emulators/86box/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, makeWrapper, freetype, SDL2
+, glib, pcre2, openal, rtmidi, fluidsynth, jack2, alsa-lib, qt5, libvncserver
+, discord-gamesdk, libpcap
+
+, enableDynarec ? with stdenv; isx86_32 || isx86_64 || isAarch64 || isAarch32
+, enableNewDynarec ? with stdenv; enableDynarec && (isAarch64 || isAarch32)
+, enableFluidsynth ? true
+, enableRtMidi ? true
+, enableVncRenderer ? false
+, enableMinitrace ? false
+, enableGdbStub ? false
+, unfreeEnableDiscord ? false
```

Comment:
Do we really need all of this option?

Reply:
Sorry, I was genuinely unsure where to draw the line, so I just included all of the configurable options. I've removed most of the options and just kept the ones that are less trivial.